examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -14,7 +14,9 @@ export interface GLTF {
 	scene: Scene;
 	scenes: Scene[];
 	cameras: Camera[];
-	asset: object;
+	asset: any;
+	parser: GLTFParser;
```

Comment:
Is the `parser` intended to be exposed?

Reply:
Yes – it's useful enough to include here, I think.